packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -552,9 +561,21 @@ export class StatsAnalyzer extends EventsScope {
       }
     });
 
+    let videoSenderIndex = 0;
     statsItem.report.forEach((result) => {
       if (types.includes(result.type)) {
-        this.parseGetStatsResult(result, type, isSender);
+        // if the video sender has multiple streams in the report, it is a new stream object.
+        if (type === 'video-send' && result.type === 'outbound-rtp') {
+          const newType = `video-send-${videoSenderIndex}`;
```

Comment:
A couple of questions here:

1. Is this the best way to label the simulcast layers? Did you consider, for example, using a nested array instead of labeling them `video-send-0`, `video-send-1`, etc.?
2. If we're going to use labels like this, could we label them `video-send-layer-0`, `video-send-layer-1` instead, to make it clear that these stats are for each simulcast layer?
3. Also, is the order deterministic? For example, would `video-send-0` _always_ be the layer with the highest frame size, or is it random?

Reply:
1. This could be done, it would require reworking quite a few parts. I'm thinking this something that could be added to the refactor ticket https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-507293
2. Good idea, updated.
3. No, it is not deterministic.